gulpfile.js

```diff
@@ -0,0 +1,122 @@
+const fs = require('fs');
+const path = require('path');
+const gulp = require('gulp');
```

Comment:
You can also make the code easier to read by importing the API's directly. 
`const { src, dest } = require('gulp');`
Then you can call `return src(E2E_PROJECT, {` without the `gulp.src`.

Reply:
Updated imports.